5/jamie-white-captions.srt

```diff
@@ -4000,7 +4003,7 @@ and maybe a poster frame.
 
 947
 01:22:50,830 --> 01:22:56,430
-JW: inaudible looks surplus to
+JW: [inaudible] looks surplus to
```

Comment:
I hear:

    That "Welcome to Peer to Peer" looks surplus to requirements now, so let's ditch that.